cmd/explaintest/r/select.result

```diff
@@ -357,19 +357,20 @@ set @@session.tidb_enable_table_partition = '1';
 create table th (a int, b int) partition by hash(a) partitions 3;
 insert into th values (0,0),(1,1),(2,2),(3,3),(4,4),(5,5),(6,6),(7,7),(8,8);
 insert into th values (-1,-1),(-2,-2),(-3,-3),(-4,-4),(-5,-5),(-6,-6),(-7,-7),(-8,-8);
+analyze table th;
 desc select * from th where a=-2;
 id	estRows	task	access object	operator info
-TableReader_7	10.00	root	partition:all	data:Selection_6
-└─Selection_6	10.00	cop[tikv]		eq(test.th.a, -2)
-  └─TableFullScan_5	10000.00	cop[tikv]	table:th	keep order:false, stats:pseudo
+TableReader_7	1.00	root	partition:all	data:Selection_6
```

Comment:
we cannot get a stable est count after change to global status that use column hist.
(before this pr it's full tablescan and can use table count directly)